chapter_preliminaries/probability_vn.md

```diff
@@ -144,7 +144,9 @@ multiple samples at once, returning an array of independent samples in any shape
 we might desire.
 -->
 
-*dịch đoạn phía trên*
+Nếu chạy bộ lấy mẫu một vài lần, bạn sẽ thấy rằng các giá trị ngẫu nhiên được tạo ra tại mỗi lần.
+Khi ước lượng tính công bằng của một con xúc xắc, chúng ta thường muốn tung nhiều lần con xúc xắc này.
```

Comment:
```suggestion
Khi ước lượng tính công bằng của một con xúc xắc, chúng ta thường muốn tung nhiều lần con xúc xắc này để tạo ra các mẫu từ cùng một phân phối.
```
Đoạn này dịch thiếu nguyên đoạn `generate many samples from the same distribution.`